examples/03-Running-on-multiple-GPUs-or-on-CPU.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<p>Can we split this into multiple cellts?</p><p> </p><p>1. Cell configurations - everything until <code>cluster = None&nbsp;</code> . we can add a description, that the code will automatically generate the parameters for the LocalCUDACLuster, such as # of GPU, memory limits, etc.</p><p>2. Cell Initialize LocalCUDACLuster - </p><p>3. Cell start the LocalCUDACluster</p><p> </p><p>Can we add a comment, that we can connect to existing Dask Cluster - using multiple nodes of NVTabular, as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>